### ISSUE TITLE
Function whose args are unknown vs Function with no args

### ISSUE THREAD

**Issue by roblabla** [reswitched/SwIPC#27]
Right now, there's no way to tell a function whose args are unknown vs a function with no arguments. I think we should either add an attribute for this (similar to `@version`, maybe `@unknownargs` ?) or, to go the C route, have function(void) denote a function without parameters, and function() denote a function whose param list we do not know (ewww).

**Comment by roblabla** [comment]
Implemented in https://github.com/reswitched/SwIPC/pull/9/commits/d28ee460aabd5ca2a88144d78347d29d73da541e

**Comment by misson20000** [comment]
Functions with no information should use `unknown` as in:
```
[0] Command(unknown) -> unknown;
```
Functions with known rawdata sizes but no further argument information should look like:
```
[0] Command(unknown<0x20>) -> unknown<0x20>;
```
Addressed in #42.

**Comment by roblabla** [comment]
Similarly, types of unknown layout but known size should use unknown<size> instead of bytes<size>, which is often used in some manual id files.
